tests/ignite/metrics/test_loss.py

```diff
@@ -27,6 +27,20 @@ def test_compute():
     assert_almost_equal(loss.compute(), 1.1253643036)  # average
 
 
+def test_compute_on_criterion():
+    loss = Loss(nn.NLLLoss())
```

Comment:
should probably add some cases where the kwargs are set

Reply:
Is there a use case where someone would have `size_average=False` and then want to pass it to the `Loss` metric? The way i was looking at it is we want to allow users to pass in the loss function the are using to train their model. I'm afraid this could become a lil pathological if we try to support every permutation. What do you think?